da/grpc/mockserv/cmd/go.mod

```diff
@@ -0,0 +1,56 @@
+module gitub.com/celestiaorg/optimint/mockda
+
+go 1.17
+
+require github.com/celestiaorg/optimint v0.2.0
```

Comment:
Hmmmmmmm. Isn't this use a specific version of `optimint`? What if we tag a new release? Then we'd hae to manually increment this _after_ release?